packages/blade/src/components/Carousel/_decisions/decisions.md

```diff
@@ -0,0 +1,176 @@
+# Carousel 
+
+Carousel is a UI component that allows the display and navigation of a set of content items, typically images or cards, within a limited space. It is often used to showcase multiple pieces of content in a visually appealing and interactive way.
+
+<img src="./carousel-thumbnail.png" width="380" />
+
+## Design
+
+[Figma Link](https://www.figma.com/file/LSG77hEeVYDk7j7WV7OMJE/Blade-DSL---Components-Guideline?type=design&node-id=2115-975416&mode=design&t=gjEux4FLvrLihxbH-0) to all variants of the Carousel component
+
+## API
+
+### Carousel
+
+| Prop                     | Type                                   | Default                                  | Description                                                                                                                                                                                                  | Required |
+| ------------------------ | -------------------------------------- | ---------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | -------- |
+| children                 | `CarouselItem`                         | `undefined`                              | Carousel slides                                                                                                                                                                                              | ✅        |
+| autoPlay                 | `boolean`                              | `false`                                  | If true, the carousel will autoplay                                                                                                                                                                          |          |
+| visibleItems             | `1,2,3,autofit`                        | `1`                                      | Total number of carousel items to show at once, can be set to 1,2,3 or if set to `autofit` the carousel items will take up 100% of the container, consumers can pass fixed width to carousel items via the carouselItemWidth prop to achive bleed.     |          |
```

Comment:
If visible items are set to 3, then how many items will slide on clicking navigation button ?

Reply:
3 items.